qemu/tests/cfg/driver_load_stress.cfg

```diff
@@ -74,6 +74,8 @@
             driver_id_pattern = "(.*?):.*?VirtIO SCSI pass-through controller"
         - with_netkvm:
             driver_name = netkvm
+            Win2016, Win2019, Win8..1, Win2012..r2:
+                extra_verifier = ndis
```

Comment:
I think you can use `driver_name += ndis`, so the VT patch is not needed.
CC @vivianQizhu

Reply:
Okay,the main reason is that `driver_name` is also used in other codes, so I have defined a new variable . Do you think this is okay? If not, I can close the avocado-vt patch and update tp-qemu patch.

Reply:
I agree with @PaulYuuu the VT patch might be closed, and I also noticed that you may not be able to directly append it to `driver_name` in cfg for some test cases, then I suppose you could step into the test case and directly append `ndis` to the parameter `driver` when calling `windrv_check_running_verifier` or other related VT functions.

Reply:
Or another solution could be, to append `ndis` to `driver_name`, and then for other usage that does not need `ndis` you might replace `params['driver_name']` with `params['driver_name'].split()[0]`.

Reply:
Or we should distinguish their usage, use a new parameter like `driver_name_verification` for `setup_win_driver_verifier`.

```
driver_name = netkvm
driver_name_verification = ${driver_name}
Win2016, Win2019, Win8..1, Win2012..r2:
    driver_name_verification += " ndis"
```

Reply:
Okay,Thank you @PaulYuuu @vivianQizhu ,will close the VT patch,and update tp-qemu patch.

Reply:
> Or we should distinguish their usage, use a new parameter like `driver_name_verification` for `setup_win_driver_verifier`.
> 
> ```
> driver_name = netkvm
> driver_name_verification = ${driver_name}
> Win2016, Win2019, Win8..1, Win2012..r2:
>     driver_name_verification += " ndis"
> ```

Better idea.

Reply:
Thanks a lot,will update the patch later.